### ISSUE TITLE
[Bug] [Accessibility] [dropdown] : Alt+arrow up doesn't work properly on docsite 

### ISSUE THREAD

**Issue by kolaps33** [microsoft/fluentui#31363]
### Library

React Components / v9 (@fluentui/react-components)

### System Info

```shell
System:
    OS: Windows 10 10.0.22631
    CPU: (12) x64 Intel(R) Xeon(R) E-2176M  CPU @ 2.70GHz
    Memory: 16.36 GB / 31.74 GB
  Browsers:
    Edge: Chromium (123.0.2420.97)
    Internet Explorer: 11.0.22621.1
```

### Are you reporting Accessibility issue?

None

### Reproduction

https://stackblitz.com/edit/nqfk2s?file=src%2Fexample.tsx

### Bug Description

Dropdown sandbox: https://stackblitz.com/edit/nqfk2s?file=src%2Fexample.tsx

##Steps 
1. Open dropdown from 
2. Navigate to some option (other then selected already)
3. Press Alt+arrow up key

## Actual Behavior
On the sandbox it select the option from list (works as expected)
On the docsite: https://react.fluentui.dev/?path=/docs/components-dropdown--default#default
it navigates to the "drawer" component 

## Expected Behavior
select the option from list

### Requested priority

Normal

### Are you willing to submit a PR to fix?

no

### Validations

- [X] Check that there isn't already an issue that reports the same bug to avoid creating a duplicate.
- [X] The provided reproduction is a minimal reproducible example of the bug.

**Comment by smhigley** [comment]
As context, this is because the Storybook platform uses `alt+down` and `alt+up` to navigate. I think it'd be a good contribution to Storybook as a whole to remove that behavior wholesale.
